### DISCUSSION TITLE
Tile orientation on loading a saved project

### DISCUSSION THREAD

**Discussion by Lewis-Cooper101** [AmyE123/TactiKit#22] [Map Editor Feedback]
So far I have used the default vesrion of the map editor (so no custom textures etc.) but I am finding it very easy to use, and rewarding to see a map design come to life so easily, even with place holders!

One thing I have noticed however is that when you load a saved project back onto the editor, the orientation of the already saved tiles reset to their defaults. For example if you rotate a castle wall 180 degrees to face another way, when you load up your project it will no longer be orientated that way. 

I'm not sure how easy this would be to fix and is only minor as you can just rotate them back, but would be nice to be able to save the orientation of tiles as well, if possible :)

**Comment by Lewis-Cooper101** [comment]
Sorry have just seen this is in the video and has been addressed

**Reply by AmyE123** [reply]
Actually I don't think I have addressed this yet 🤔
